apps/dashboard/src/components/subscribers/preferences/subscriber-preferences.tsx

```diff
@@ -0,0 +1,135 @@
+import { showSuccessToast } from '@/components/primitives/sonner-helpers';
```

Comment:
Naming suggestion: `apps/dashboard/src/components/subscribers/subscriber-global-preferences.tsx`

Reply:
this component contains both global and workflow preferences